src/helpers/helpers.collection.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+// @see adapted from https://www.anujgakhar.com/2014/03/01/binary-search-in-javascript/
+export function lookup(table, key, value) {
```

Comment:
I'd probably make this private for now

Reply:
done

Reply:
Did you push it? It doesn't look like it starts with an underscore yet

Reply:
Forgot the underscore. Added.